sample-apps/javascript-sample-app/random-metrics.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS'" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+create_cfg = require('./config');
+
+const { CollectorMetricExporter } = require('@opentelemetry/exporter-collector-grpc');
+const { MeterProvider } = require('@opentelemetry/metrics');
+const { Resource } = require('@opentelemetry/resources');
+const { SemanticResourceAttributes } = require('@opentelemetry/semantic-conventions');
```

Comment:
I'm not sure these are required anymore now that you're bringing in the meter from `./meter`.

Reply:
thanks. Updated.